tls/s2n_tls13.h

```diff
@@ -23,15 +23,8 @@
 extern "C" {
 #endif
 
-#if S2N_GCC_VERSION_AT_LEAST(4, 5, 0)
-    S2N_API
-    __attribute__((deprecated("The use of TLS1.3 is configured through security policies")))
-    extern int s2n_enable_tls13();
-#else
-    S2N_API
-    __attribute__((deprecated))
-    extern int s2n_enable_tls13();
-#endif
+S2N_DEPRECATED("The use of TLS1.3 is configured through security policies")
+extern int s2n_enable_tls13();
```

Comment:
This still needs to be marked S2N_API so that it's visible-- some customers still call it.